policy_packs/azure/storage/enforce_creator_timestamp_tags_for_storage_accounts/policies.tf

```diff
@@ -0,0 +1,46 @@
+# Azure > Storage > Storage Account > Tags
+resource "turbot_policy_setting" "azure_storage_storage_account_tags_detective" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/azure-storage#/policy/types/storageAccountTags"
+  value    = "Check: Tags are correct"
+  # value    = "Enforce: Set tags"
+}
+
+# Azure > Storage > Storage Account > Tags > Template
+resource "turbot_policy_setting" "azure_storage_storage_account_tags_template_detective" {
+  resource       = turbot_policy_pack.main.id
+  type           = "tmod:@turbot/azure-storage#/policy/types/storageAccountTagsTemplate"
+  template_input = <<-EOT
+    {
+      resource {
+        turbot {
+          title
+          tags
+        }
+      creator: notifications(filter: "sort:version_id limit:1") {
```

Comment:
I think we should use the createTimestamp and createdBy metadata for the storage account.